### DISCUSSION TITLE
How do you point an e2e test to an external (or running) server?

### DISCUSSION THREAD

**Discussion by matthew-dean** [nuxt/nuxt#26629] [Questions]
[This page](https://nuxt.com/docs/getting-started/testing#end-to-end-testing) describes setting up e2e testing, with the assumption that one will always want to:
1. start a new server
2. have that server built from local source

What if you just want to run tests against a running or external (remote) server? The page doesn't mention it. Even if you set `server: false` it builds / runs a production or dev server.

**Comment by TheAlexLichter** [comment]
You could use the [env-aware config](https://www.youtube.com/watch?v=DFZI2iVCrNc) and set the baseUrl of your API or variable differently in `$test` so it will point towards it ☺️

**Reply by matthew-dean** [reply]
Oh interesting! That's useful, although... I think my issue is that Nuxt config is being read at all? In other words, yeah, I want Nuxt config to be used locally / in dev mode to test my tests. But I want the ability to run Cucumber / Playwright against ANY Nuxt build anywhere, whether that Nuxt app is local or built and deployed remotely. I recognize I could just use Playwright / Cucumber without using the Nuxt integration, but then I want the custom page events to wait for page hydration / Nuxt url resolution etc.
